lib/tapioca/loaders/dsl.rb

```diff
@@ -85,7 +85,12 @@ def load_dsl_extensions
       def load_dsl_compilers
         say("Loading DSL compiler classes... ")
 
-        # Load built-in compilers and custom compilers exported from gems
+        # Load all built-in compilers
```

Comment:
If you want, we can be more explicit in this comment, so that future onlookers will not be confused.